### PR TITLE
First commit

### PR DESCRIPTION
**Pull request by p3tromyz0n**

### CODE REVIEW

**Comment by umed48** [comment]
at first, you have too many white spaces. You have to use pylint, it's really helpful and easy, try it 👍

File: logpass_line.txt [hunk 1]
```diff
@@ -0,0 +1,13 @@
+stas aaaaa
+
```
Comment (umed48) [inline logpass_line.txt]: remove redundant spaces, we want to have structured database 🔢

File: test_search.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import re
+
+
+# create function which searches login and password in file, login and password are gotten from user input
```
Comment (umed48) [inline test_search.py]: Commenting your code is good idea, we will discuss it tomorrow and bring some changes

File: test_search.py [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+def search_log_pass_in_file(login=input("Insert login: "), passw=input("Insert password: ")):
```
Comment (umed48) [inline test_search.py]: don't read user input as value of function parameters, it's bad practice. You should do it before calling this function

File: logpass_line.txt (hunk 1 above)
Comment (p3tromyz0n) [reply logpass_line.txt]: Done

File: test_search.py (hunk 2 above)
Comment (p3tromyz0n) [reply test_search.py]: Corrected

File: test_search.py (hunk 3 above)
Comment (p3tromyz0n) [reply test_search.py]: Corrected

**Pull request closed without merging**